src/inputs/internal/ListBox.tsx

```diff
@@ -84,7 +84,7 @@ export function ListBox<O, V extends Key>(props: ListBoxProps<O, V>) {
           // We don't really need to set this, but it's handy for tests, which would
           // otherwise render just 1 row. A better way to do this would be to jest.mock
           // out Virtuoso with an impl that just rendered everything, but doing this for now.
-          initialItemCount={3}
+          initialItemCount={5}
```

Comment:
**question:** Why did this change?

Reply:
Tests only render this number of items. The test I was working on needed access to more items in Virtuoso.